code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -18,6 +18,7 @@
 	force = 200
 	armour_penetration = 100
 	anchored = TRUE
+	flags = HANDSLOW
```

Comment:
The other items I looked (like glass shards) just have CONDUCTIVE and nothing else, so I dont think there are any base level flags items need.